clio-integration-test/src/main/python/clio_test.py

```diff
@@ -215,6 +215,44 @@ def expected():
     response = requests.get('/'.join([clio_http_uri, 'readgroup', 'schema', 'v1']))
     assert response.json() == expected()
 
+def test_query_sample_project():
+    version = 'v1'
+    project = 'testProject' + new_uuid()
+    upserts = [{
+            'flowcell_barcode': 'barcode2',
+            'lane': 2,
+            'library_name': 'library' + new_uuid(),
+            'location': 'GCP',
+            'project': project
+        } for _ in range(3)]
+
+    sample1 = 'testSample' + new_uuid()
+    sample2 = 'testSample' + new_uuid()
+    upserts[0]['sample_alias'] = sample1
+    upserts[1]['sample_alias'] = sample1
+    upserts[2]['sample_alias'] = sample2
+
+    for record in upserts:
+        upsertUri = '/'.join([clio_http_uri, 'readgroup', 'metadata', version,
```

Comment:
(Not your fault) This URI and the one beneath need to include an "api" path segment before "readgroup", like is happening in #50.